.idea/shelf/Uncommitted_changes_before_Checkout_at_25_02_2021_00_27_[Default_Changelist]/shelved.patch

```diff
@@ -0,0 +1,37 @@
+Index: .idea/workspace.xml
```

Comment:
it's better to remove this files from git at all

Reply:
it's just add in gitignor (*xml, *iml) ?

Reply:
Is it just add in gitignor (*xml, *iml) ?